src/async.rs

```diff
@@ -389,8 +389,17 @@ impl AsyncClient {
 
     /// Get transaction history for the specified address/scripthash, sorted with newest first.
     /// Returns up to 50 mempool transactions plus the first 25 confirmed transactions.
-    pub async fn get_address_txns(&self, address: &Address) -> Result<Vec<Tx>, Error> {
-        let path = format!("/address/{address}/txs");
+    /// You can request more confirmed transactions using an after_txid parameter.
+    pub async fn get_address_txns(
+        &self,
+        address: &Address,
+        after_txid: Option<Txid>,
+    ) -> Result<Vec<Tx>, Error> {
+        let path = match after_txid {
+            Some(after_txid) => format!("/address/{address}/txs/chain/{after_txid}"),
+            None => format!("/address/{address}/txs"),
+        };
+
```

Comment:
As they're somewhat twin endpoints, I'd suggest following the `scripthash_txs`, and (1) updating it to `address_txs` and (2) using `last_seen` instead of `after_txid`.

Also, the documentation could be updated accordingly.

Reply:
Yup I’ll do that

Reply:
done